tests/dataimportcron_test.go

```diff
@@ -512,9 +512,11 @@ var _ = Describe("DataImportCron", Serial, func() {
 			_, err = f.K8sClient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Update(context.TODO(), pvc, metav1.UpdateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			pvcList, err := f.K8sClient.CoreV1().PersistentVolumeClaims(ns).List(context.TODO(), metav1.ListOptions{})
-			Expect(err).ToNot(HaveOccurred())
-			Expect(pvcList.Items).To(HaveLen(garbageSources + 1))
+			Eventually(func() int {
+				pvcList, err := f.K8sClient.CoreV1().PersistentVolumeClaims(ns).List(context.TODO(), metav1.ListOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return len(pvcList.Items)
+			}, dataImportCronTimeout, pollingInterval).Should(Equal(garbageSources + 1))
```

Comment:
Why are we switching to eventually here?

Reply:
The flakiness was because sometime `pvcList` still included a pvc-prime which was not deleted yet. `Eventually` fixes this flakiness.

Reply:
oh okay cool. consider using `.Should(HaveLen(x))` here

Reply:
fixed